documentation/dotnet-dump-instructions.md

```diff
@@ -17,7 +17,7 @@ Here's a table showing how dotnet-dump fits into your dump debugging options:
 
 ## Installing dotnet-dump
 
-The first step is to install the dotnet-dump CLI global tool. This requires at least the 2.1 or greater .NET Core SDK to be installed. If you see the error message `Tool 'dotnet-dump' is already installed`, you will need to uninstall the global tool (see below). 
+The first step is to install the dotnet-dump CLI global tool. Installing `dotnet-dump` requires at least the 2.1 or greater .NET Core SDK to be installed.  Note, that _collecting_ a dump on non-Windows platforms requires the target application to run on a 3.0 or newer .NET Runtime. If you see the error message `Tool 'dotnet-dump' is already installed`, you will need to uninstall the global tool (see below). 
```

Comment:
I know this means dotnet-dump only runs on non Windows platform, but to someone who's reading it for the first time it might lead them to think collecting it on Windows works and it works for versions less than 3.0 too.

Reply:
Above where we have the table showing options for analysis I think we need another table for collection (on OS _x_ you need runtime version >= _y_). Then here you can say "Using dotnet-dump to collect dumps also requires support from the .Net Core runtime in the target process. See the \[table\](some_link) above."